src/unix/linux_like/emscripten/mod.rs

```diff
@@ -1784,6 +1784,10 @@ extern "C" {
     pub fn srand(seed: ::c_uint);
 
     pub fn gettimeofday(tp: *mut ::timeval, tz: *mut ::c_void) -> ::c_int;
+    pub fn clock_getcpuclockid(
+        pid: ::pid_t,
+        clk_id: *mut ::clockid_t,
+    ) -> ::c_int;
```

Comment:
It should be available on Android as well, let's move it to `linux_like/mod.rs`.

Reply:
I originally tried to put it in `linux_like/mod.rs`, but the build for android failed due to `clock_getcpuclockid` being missing.

Reply:
If it's on aarch64 only, then the cause is not absence, but #1765 likely.

Reply:
Ah, yes, it was [aarch64](https://dev.azure.com/rust-lang2/libc/_build/results?buildId=2966&view=logs&j=c8d582de-107e-5fbd-122b-5384eb107855&t=7bb3b962-ee23-58e7-68da-769fcc260c0d) . In that case - should I keep it as is for now and move it to `linux_like/mod.rs` once the issue is resolved?

Reply:
I'd like to skip the tests for now, could you tweak `libc-test/build.rs` to do so? (see https://github.com/rust-lang/libc/pull/1855/files#diff-747c41e91bd79d9208c2953b72047176 for example.)

Reply:
Thanks for the example, done!